src/style.css

```
@@ -1 +1,79 @@
-/* Styles go here */
+:root {
+  --color-font: #000;
+  --color-active: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", 'Times New Roman', sans-serif;
+  color: var(--color-font);
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 1200px;
+  padding: 0;
+  margin: auto;
```

```suggestion
  padding: 0 50px;
```
it is should fix tests